shotover-proxy/tests/test-configs/cassandra-peers-rewrite/docker-compose-3.11-cassandra.yaml

```diff
@@ -44,3 +46,14 @@ services:
     environment: *environment
     depends_on:
       - cassandra-one
+
+  cassandra-three:
```

Comment:
Why have we introduced a third cassandra instance here?

Reply:
So we can kill it to test the events.

Reply:
Oh I see, makes sense :+1: